scripts/deploy.ts

```diff
@@ -38,15 +38,22 @@ async function main() {
   // Create health bot table
   const { chainId } = await account.provider.getNetwork();
   const createStatement = `create table healthbot_${chainId} (counter integer);`;
-  let tx = await tables.createTable(account.address, createStatement);
+  let tx = await tables["runSQL(address,string)"](
```

Comment:
@sanderpick @carsonfarmer  when using overloads the ts/js ends up looking like this.  I don't have a super strong opinion, thoughts?

Reply:
This is no longer the case in Ethers.js v6.  Once that lands in Hardhat we can upgrade here, and this api goes back to the nicer looking `tables.createTable(...)`